sigmastate/src/main/scala/sigmastate/lang/exceptions/SigmaBinderExceptions.scala

```diff
@@ -5,5 +5,3 @@ import sigmastate.lang.SourceContext
 final class InvalidArguments(message: String, source: Option[SourceContext] = None)
```

Comment:
Rename the file to "InvalidArguments.scala"

Reply:
done